microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,18 +33,29 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.
 
 > [!NOTE]
 > The **Publish APIs** UI is not yet present at Partner Center.
 
-1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
+1. Visit Microsoft Partner Center and sign in to the account from which you have already published an add-on.
 
-1. Under the **Microsoft Edge** program, select **Publish APIs**.
+1. Under the **Microsoft Edge** program, select **Publish API**.
 
-1. In the **Publish APIs** page, select the **Create API credentials** button to generate the API credentials.  This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
+1. In the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes.
 
-1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**.  You'll use them in the next step to get an access token.
+   :::image type="content" source="../../media/create-api-credentials-button.png" alt-text="The Publish API page at the Microsoft Partner Center showing the Create API credentials button." lightbox="../../media/create-api-credentials-button.png":::
+
+   <!-- make window smaller before capture.  make fresh capture.  save .png in present repo as ../media/create-api-credentials-button.png, replacing path in parens -->
+
+   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client Secret**, and **Auth Token URL** are now displayed on the Publish APIs page.
```

Comment:
```suggestion
   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page.
```

Reply:
See the image in the rendered page, to check wording about "access" vs. "auth".  @vchapel  
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1694#before-you-begin